### PR TITLE
Add `allowed_bastion_urls` to chart

### PR DESCRIPTION
**Pull request by rani-sharim**
Add `allowed_bastion_urls` to chart

### CODE REVIEW

**Review by omriezra — APPROVED** [review]

File: charts/akeyless-secure-remote-access/README.md [hunk 1]
```diff
@@ -145,6 +145,7 @@ The following table lists the configurable parameters of the SSH Bastion chart a
 | `sshConfig.readinessProbe`                          | Readiness probe configuration for SSH Bastion                                                                        | Check `values.yaml` file                                     |         
 | `sshConfig.resources.limits`                        | The resources limits for SSH Bastion containers  (If HPA is enabled these must be set)                               | `{}`                                                         |
 | `sshConfig.resources.requests`                      | The requested resources for SSH-Bastion containers (If HPA is enabled these must be set)                             | `{}`                                                         |
+| `sshConfig.allowedBastionUrls` | List of URLs that will be used for automatic sra resolution       | `[]`                          |
```
Comment (OriMankali) [inline charts/akeyless-secure-remote-access/README.md:148]: Please pass the text through product for review. What about Proxy URL?

File: charts/akeyless-secure-remote-access/templates/statefulSet.yaml [hunk 2]
```diff
@@ -108,6 +108,10 @@ spec:
           - name: ALLOWED_ACCESS_IDS
             value: {{ range $index, $element := .Values.privilegedAccess.allowedAccessIDs -}}{{- if $index -}},{{- end -}}{{ $element }}{{ end }}
 {{- end }}
+{{- if .Values.sshConfig.allowedBastionUrls }}
```
Comment (OriMankali) [inline charts/akeyless-secure-remote-access/templates/statefulSet.yaml:111]: What about ZTB?

File: charts/akeyless-secure-remote-access/README.md (hunk 1 above)
Comment (rani-sharim) [reply charts/akeyless-secure-remote-access/README.md]: Talked to Eldad, he asked to set it to be the same as the rest.
Also added allowedProxyUrls

File: charts/akeyless-secure-remote-access/templates/statefulSet.yaml (hunk 2 above)
Comment (rani-sharim) [reply charts/akeyless-secure-remote-access/templates/statefulSet.yaml:111]: Ztb had this for a while now

**Review by OriMankali — APPROVED** [review]

**Pull request merged**